Data/Root/RootDataIO.cpp

```diff
@@ -21,116 +23,115 @@ namespace ComPWA {
 namespace Data {
 namespace Root {
 
-RootDataIO::RootDataIO(const std::string TreeName_,
-                       std::size_t NumberEventsToProcess_)
-    : TreeName(TreeName_), NumberEventsToProcess(NumberEventsToProcess_) {}
-
-std::vector<ComPWA::Event>
-RootDataIO::readData(const std::string &InputFilePath) const {
-  /// @todo Use of `TChain` in this way may result in run-time errors for larger
-  /// data samples.
-  TChain chain(TreeName.c_str());
-  chain.Add(InputFilePath.c_str());
-  if (!chain.GetListOfFiles()->GetEntriesFast())
-    throw std::runtime_error("RootDataIO::RootDataIO() | "
-                             "Unable to load files: " +
-                             InputFilePath);
+std::vector<ComPWA::Event> readData(const std::string &InputFilePath,
+                                    const std::string &TreeName,
+                                    long long NumberOfEventsToRead) {
+  // Ignore custom streamer warning and error message for missing trees
+  gErrorIgnoreLevel = kFatal;
```

Comment:
This also ignores fatal errors?

Reply:
I increased the level a bit now, to `kBreak`, but the above means that  `kFatal` is still shown.